### ISSUE TITLE
unknown member of modules with numpy and Import "gnuradio" could not be resolved

### ISSUE THREAD

**Issue by njse22** [microsoft/pyright#1485]
**Describe the bug**
Hi all, I recently start to use pyright and it is amazing but I have a little 'bug' withe the syntax of some modules of numpy and a problem and a problem with import of particular library (gnuradio) 

**To Reproduce**
with numpy some modules marks like a error, for example:  
```python
import numpy as np 

x = np.random.randn(1000)*15+100 # show -> "randn" is not a know member of module [Pyright: reportGeneralTypeIssues] 

in_sig=[np.complex64] # show -> "complex64" is not a known member of module [Pyright: reportGeneralTypeIssues]

x2 = np.arange(-1,1,0.1)
sig = np.sin(x) # show -> "sin" is not a known member of module [Pyright: reportGeneralTypeIssues], and this is the same output for cos(x), tan(x), arcsin(x)... 
```
And with [gnuradio](https://github.com/gnuradio/gnuradio) 

```python 
from gnuradio import gr # show -> Import "gnuradio" could not be resolved [Pyright: reportMissingImports]
```

**Additional context**
I use nvim version: 0.4.4 with coc-pyright

**Comment by erictraut** [comment]
I presume you're using numpy 1.20?

The symbol `complex64` is not declared in numpy's `__init__.pyi` file, so pyright is correct in flagging it as an unknown member of the module. If you believe this symbol should be declared, please file a bug or a PR in the numpy github repo.

I don't know anything about the "gnuradio". I couldn't find anything by that name on pypi. The error you are seeing from pyright indicates that the package couldn't be resolved in your configured python environment.

**Comment by njse22** [comment]
no, I just checked and I have version 1.13.3.... I guess that's the problem with numpy, with gnuradio ... I don't know jedi-vim doesn't show me the same error, but maybe it's also a configuration error of mine.

**Comment by erictraut** [comment]
If you're using an old version of numpy (prior to 1.20), then there's no type information in the numpy package. They just added that in 1.20. Perhaps you have separate type stubs installed for numpy? If so, then pyright is getting the type information from those stubs, and you would want to report the missing symbol to the authors of those stubs.

Keep in mind that pyright's type checking is only as good as the type information provided to it. Many library authors are just starting to get onboard with distributing type information within their packages.
